### PR TITLE
Quizbackend

### PR DESCRIPTION
**Pull request by shreyshrey1**
### Test plan
There's no dummy data but I think it works

Query:   
```
lessons {
  	quiz {
  	  id
          name
          questions {
            questionName
            answers {
              answerName
              isCorrect
           }
         }
       }
     }
   }
```
Result:
```
{
  "data": {
    "lessons": [
      {
        "quiz": {
          "id": "UXVpejo=",
          "name": null,
          "questions": null
        }
      },
      {
        "quiz": {
          "id": "UXVpejo=",
          "name": null,
          "questions": null
        }
      }
    ]
  }
}
```

### CODE REVIEW

**Comment by tko22** [comment]
just a tip: use ``` to wrap around code, one ``` in the beginning and another ``` at the end.

File: web_backend/src/models/lessons.js [hunk 1]
```diff
@@ -1,27 +1,23 @@
-const mongoose = require('mongoose');
+import mongoose from 'mongoose'
```
Comment (tko22) [inline web_backend/src/models/lessons.js]: semi colon. Use eslint's `fix all auto-fixable problems` before you commit.

File: web_backend/src/models/lessons.js [hunk 2]
```diff
@@ -1,27 +1,23 @@
...
-    //   }
+    quiz: {
+      name: { type: String },
+      questions: [{
+          questionName: { type: String },
+          answers: [{
+              answerName: { type: String },
+              isCorrect: { type: Boolean }
+          }]
+      }],
```
Comment (tko22) [inline web_backend/src/models/lessons.js]: good formatting

**Comment by CodeBrew28** [comment]
You should add data using other mutations so that you can fully test it
also 3 back ticks before and after allows you to format your code 
```
hi
```

**Comment by tko22** [comment]
I believe this was done in #52

**Pull request closed without merging**